src/interface/variable.hpp

```diff
@@ -159,6 +159,7 @@ class CellVariable {
 /// and label so that we can refer to variables by name.  Since Athena
 /// currently only has scalar Face fields, we also only allow scalar
 /// face fields
+//  FIXME(forrestglines): What is it meant by only "scalar" Face fields? Athena++ ONLY has vector face fields (x,y,z for each face)
```

Comment:
I'm not entirely sure how face-centered tensor-valued fields would be used since I'm not entirely sure how cell-centered tensor-valued fields are used in downstream methods.

The only example I can imagine is an two-fluid MHD method I like to implement where magnetic and electric fields are co-located at the faces and evolved together.

Reply:
Any tensor where one index is divergence free---for example a stress tensor---would be appropriate as a face-centered value. That said, I don't know if we have such a thing at the moment and I wouldn't worry about it too much.

Reply:
I think it is probably useful to have tensor fields definable at faces, etc. For instance, one might want the three-metric on a face. I also don't think there is anything limiting us from doing this.